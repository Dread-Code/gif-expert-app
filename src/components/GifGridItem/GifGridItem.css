.card {
  border-radius: 0.2em;
  background-position: center;
  align-content: center;
  background-repeat: no-repeat;
  background-size: cover;
}
