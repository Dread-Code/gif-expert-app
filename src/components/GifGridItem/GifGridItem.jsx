import React from 'react'
import './GifGridItem.css'
import PropTypes from 'prop-types'

const GifGridItem = ({ title, url, height, width }) => {
  return (
    <div
      className="card animate__animated animate__fadeIn"
      style={{ backgroundImage: `url("${url}")` }}
      alt={title}
    />
  )
}

GifGridItem.propTypes = {
  title: PropTypes.string.isRequired,
  url: PropTypes.string.isRequired
}

export default GifGridItem
