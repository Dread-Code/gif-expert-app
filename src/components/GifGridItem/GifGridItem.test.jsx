import { shallow } from 'enzyme'
import GifGridItem from './GifGridItem'

describe('Gif Grid Item', () => {
  test('should Show shallow', () => {
    const wrapper = shallow(<GifGridItem />)

    expect(wrapper).toMatchSnapshot()
  })
})
