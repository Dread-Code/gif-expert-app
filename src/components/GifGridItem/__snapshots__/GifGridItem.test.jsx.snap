// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gif Grid Item should Show shallow 1`] = `
<div
  className="card animate__animated animate__fadeIn"
  style={
    Object {
      "backgroundImage": "url(\\"undefined\\")",
    }
  }
/>
`;
